### ISSUE TITLE
Map values not shown correctly in the interpreter

### ISSUE THREAD

**Issue by joey-coleman** [symphonytool/symphony#115]
> The following bug was [originally reported](https://sourceforge.net/p/compassresearch/bugs/115/) on Sourceforge by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-10-10 11:50:46.550000:

We dont show map values correctly. They cannot be expanded like sequences.

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/115/#86a3) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-18 15:56:06.814000:
- Description has changed:

Diff:

```

--- old
+++ new
@@ -1 +1 @@
-We dont show map values correctly. Then cannot be expanded like sequences.
+We dont show map values correctly. They cannot be expanded like sequences.

```
- **status**: open --> accepted
- **Module**:  --> interpreter

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/115/#5b43) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-20 14:49:14.757000:

fixed in [8303964efe5fccab0535570fd2487b23cf2544bd]

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/115/#ac9b) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-20 14:49:31.176000:
- **status**: accepted --> closed
